batch/batch/globals.py

```diff
@@ -17,4 +17,4 @@
 HTTP_CLIENT_MAX_SIZE = 8 * 1024 * 1024
 
 BATCH_FORMAT_VERSION = 2
-INSTANCE_VERSION = 2
+INSTANCE_VERSION = 3
```

Comment:
Can you remind me what INSTANCE_VERSION versions?  This doesn't seem to be used now.

It is the case that this changes the status format.  Should we have a status version, and put the version in the json.  Consumers can use the version to figure out the schema and interpret the status.  We should start with v2.  No version means v1.

I think the rule going forward is anything that's stored needs to be versioned.

Reply:
The instance_version was mainly used so we could tell in the UI when we had drained the worker pool before merging breaking changes. I agree we should store the version in the status, but I don't think we need to do anything else.

Reply:
Sorry, I didn't mean 'v2' literally, I just meant version 2.  I think we should stay with integer versions, unless we need semantic versioning.